service/service.go

```diff
@@ -254,7 +276,7 @@ func New(config Config) (*Service, error) {
 		frameworkConfig.BackOff = frameworkBackOff
 		frameworkConfig.InitCtxFunc = initCtxFunc
 		frameworkConfig.Logger = config.Logger
-		frameworkConfig.Resources = resources
+		frameworkConfig.ResourceRouter = framework.NewDefaultResourceRouter(resources)
```

Comment:
Here we use the new ResourceRouter. It's not fully implemented yet so all resources are returned.